src/libraries/System.Runtime/tests/TrimmingTests/System.Runtime.TrimmingTests.proj

```diff
@@ -19,12 +19,6 @@
       <EnabledFeatureSwitches>System.Globalization.Invariant;System.Globalization.PredefinedCulturesOnly</EnabledFeatureSwitches>
     </TestConsoleAppSourceFiles>
     <TestConsoleAppSourceFiles Include="StackFrameHelperTest.cs">
-      <!-- There is a bug with the linker where it is corrupting the pdbs while trimming
```

Comment:
This is a good change, but orthogonal to renaming (it is a functional change). In general it's preferable not to do functional changes in big PRs like this.